### ISSUE TITLE
500 Internal Server Error when updating HACS

### ISSUE THREAD

**Issue by Pirol62** [hacs/integration#753]
**Home Assistant release with the issue:**
0.100.2

**Last working Home Assistant release (if known):**

0.100.2
**Operating environment (Hass.io/Docker/Windows/etc.):**

**Integration:**

**Description of problem:**

opening the community page I got this picture
![image](https://user-images.githubusercontent.com/48820003/67888595-b4730080-fb4d-11e9-8569-9add37bd6b38.png)

After upgrade i got 500 Internal Server error

**Problem-relevant `configuration.yaml` entries and (fill out even if it seems unimportant):**
```yaml

```

**Traceback (if applicable):**
```
Wed Oct 30 2019 19:41:35 GMT+0100 (Mitteleuropäische Normalzeit)
Error handling request
Traceback (most recent call last):
  File "/config/custom_components/hacs/handler/download.py", line 36, in async_download_file
  File "/usr/local/lib/python3.7/site-packages/aiohttp/client_reqrep.py", line 973, in read
    self._body = await self.content.read()
  File "/usr/local/lib/python3.7/site-packages/aiohttp/streams.py", line 358, in read
    block = await self.readany()
  File "/usr/local/lib/python3.7/site-packages/aiohttp/streams.py", line 380, in readany
    await self._wait('readany')
  File "/usr/local/lib/python3.7/site-packages/aiohttp/streams.py", line 296, in _wait
    await waiter
concurrent.futures._base.CancelledError

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "/config/custom_components/hacs/api.py", line 43, in post
  File "/config/custom_components/hacs/api.py", line 168, in response
  File "/config/custom_components/hacs/repositories/repository.py", line 384, in install
  File "/config/custom_components/hacs/repositories/repository.py", line 456, in download_zip
  File "/usr/local/lib/python3.7/site-packages/backoff/_async.py", line 131, in retry
    ret = await target(*args, **kwargs)
  File "/config/custom_components/hacs/handler/download.py", line 40, in async_download_file
  File "/usr/local/lib/python3.7/site-packages/async_timeout/__init__.py", line 45, in __exit__
    self._do_exit(exc_type)
  File "/usr/local/lib/python3.7/site-packages/async_timeout/__init__.py", line 92, in _do_exit
    raise asyncio.TimeoutError
concurrent.futures._base.TimeoutError

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "/usr/local/lib/python3.7/site-packages/aiohttp/web_protocol.py", line 418, in start
    resp = await task
  File "/usr/local/lib/python3.7/site-packages/aiohttp/web_app.py", line 458, in _handle
    resp = await handler(request)
  File "/usr/local/lib/python3.7/site-packages/aiohttp/web_middlewares.py", line 119, in impl
    return await handler(request)
  File "/usr/src/homeassistant/homeassistant/components/http/real_ip.py", line 40, in real_ip_middleware
    return await handler(request)
  File "/usr/src/homeassistant/homeassistant/components/http/ban.py", line 73, in ban_middleware
    return await handler(request)
  File "/usr/src/homeassistant/homeassistant/components/http/auth.py", line 231, in auth_middleware
    return await handler(request)
  File "/usr/src/homeassistant/homeassistant/components/http/view.py", line 128, in handle
    result = await result
  File "/config/custom_components/hacs/api.py", line 45, in post
  File "/config/custom_components/hacs/http.py", line 98, in render
  File "/usr/local/lib/python3.7/site-packages/jinja2/environment.py", line 830, in get_template
    return self._load_template(name, self.make_globals(globals))
  File "/usr/local/lib/python3.7/site-packages/jinja2/environment.py", line 804, in _load_template
    template = self.loader.load(self, name, globals)
  File "/usr/local/lib/python3.7/site-packages/jinja2/loaders.py", line 113, in load
    source, filename, uptodate = self.get_source(environment, name)
  File "/usr/local/lib/python3.7/site-packages/jinja2/loaders.py", line 235, in get_source
    raise TemplateNotFound(template)
jinja2.exceptions.TemplateNotFound: error.html
```

**Additional information:**
``

**Comment by ludeeus** [comment]
Do you have a HACS folder?

**Comment by Pirol62** [comment]
> Do you have a HACS folder?

No, there is no folder

**Comment by ludeeus** [comment]
Same issue as https://github.com/hacs/integration/issues/734 then.

Download it again and it will be good.
https://hacs.xyz/docs/installation/manual

**Comment by Pirol62** [comment]
ok, I restored my backup and everything seems to be normal.
I did not restart so far.
Open questions:
1. duckdns doesn't work since 15 Minutes (Exception while resolving host: (12, b'Timeout while contacting DNS servers')
) - has it something to do with this issue?
2. shall I upgrade hacs again? Before or after a HA restart?
3. why do I still have two entriys of HACS in the overview?

**Comment by ludeeus** [comment]
1: Not relevant, but you should probably fix that too (I do not know how)
2: Before, manually, with the link I gave you._
3: Fixed in 0.16.X
